airbyte-local-cli-nodejs/src/index.ts

```diff
@@ -0,0 +1,12 @@
+import {parseAndValidateInputs} from './command';
+import {logger} from './utils';
+
+async function main() {
+  logger.debug('Starting Airbyte Local CLI...');
+  await parseAndValidateInputs(process.argv);
```

Comment:
passing in `process.argv` explicitly for easier testing. 
otherwise commander can implicitly parse this as well.